packages/terra-search-field/src/terra-dev-site/doc/search-field/SearchField.1.doc.jsx

```diff
@@ -92,6 +94,11 @@ const DocPage = () => (
         example: <SearchFieldFocus />,
         source: SearchFieldFocusSrc,
       },
+      {
+        title: 'Search Field with setting a value programmatically',
+        example: <SearchFieldProgrammaticSet />,
+      source: SearchFieldProgrammaticSetSrc,
```

Comment:
do we need a first class example for consumers?

Reply:
Since there is a bit of complexity getting the expected functionality out of programmatically setting a value because of the need to create a custom 'input' event dispatch, @dkasper-was-taken and I wanted an example.